File: test/three/app/src/main/java/com/z3d/skill/key/SkillFixEffect.java
package com.z3d.skill.key;

import com.z3d.skill.vo.SkillFixEffectKeyVo;
import com.z3d.skill.vo.SkillKeyVo;
import com.z3d.vo.Vector3D;

public class SkillFixEffect extends SkillEffect {


    public Vector3D pos;
    public Vector3D rotation;
    public Vector3D outPos  ;
    public boolean hasSocket;
    public String socket;

    public void setInfo(SkillKeyVo skillKeyVo) {
        super.setInfo(skillKeyVo);
        SkillFixEffectKeyVo data   =(SkillFixEffectKeyVo) skillKeyVo;
        this.pos = data.pos;
        this.rotation = data.rotation;
        this.hasSocket = data.hasSocket;
        this.socket = data.socket;
    }
}

File: metalios/iosgl/iosgl/bt/skill/key/SkillFixEffect.m
//
//  SkillFixEffect.m
//  iosgl
//
//  Created by zhao on 17/4/2020.
//  Copyright © 2020 zhao. All rights reserved.
//

#import "SkillFixEffect.h"
#import "SkillFixEffectKeyVo.h"

@implementation SkillFixEffect

- (void)setInfo:(SkillKeyVo *)obj
{
    [super setInfo:obj];
    SkillFixEffect* this=self;
    SkillFixEffectKeyVo* data = (SkillFixEffectKeyVo*)obj;
    this.pos = data.pos;
    this.rotation = data.rotation;
    this.hasSocket = data.hasSocket;
    this.socket = data.socket;
    
}

 

@end
